samples/net/cloud/aws_iot_mqtt/boards/ek_ra8m1.conf

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ENTROPY_DEVICE_RANDOM_GENERATOR=y
+
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_DHCPV4_INITIAL_DELAY_MAX=2
```

Comment:
is it necessary to set this to 2 instead using the default of 10?

Reply:
Thank you, I rebase the change and remove the commit for this.